specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -2149,23 +2159,31 @@
       "properties": {
         "rotationEnabled": {
           "type": "boolean",
+          "required": false,
```

Comment:
Nit: the default for `required` is `false` so this isn't necessary, but doesn't hurt.